test/LSM/pond_soil_lsm.jl

```diff
@@ -5,9 +5,8 @@ using ClimaLSM.Domains: LSMMultiColumnDomain, LSMSingleColumnDomain
 using ClimaLSM.Soil
 using ClimaLSM.Pond
 
-FT = Float64
 @testset "Pond soil Multi Column LSM integration test" begin
-
+    FT = Float64
```

Comment:
Maybe we should test for Float32 and Float64?

Reply:
We could. what about doing it for Float32 and making sure the type of the output is Float32? Then we know there is no promotion. 

since this isnt related to this PR, I think I will leave alone for now. But Ill open an issue to revisit the unit tests and make sure they are testing the different FT capability comprehensively.